src/Softeq.NetKit.Chat.SignalR/Hubs/Notifications/ChannelNotificationService.cs

```diff
@@ -34,20 +34,20 @@ public async Task OnUpdateChannel(ChannelSummaryResponse channel)
             await HubContext.Clients.Clients(clientIds).SendAsync(HubEvents.ChannelUpdated, channel);
         }
 
-        public async Task OnCloseChannel(ChannelSummaryResponse channel)
+        public async Task OnCloseChannel(Guid channelId)
         {
-            var clientIds = await GetChannelClientConnectionIdsAsync(channel.Id);
+            var clientIds = await GetChannelClientConnectionIdsAsync(channelId);
 
             // Tell the people in this room that you've joined
-            await HubContext.Clients.Clients(clientIds).SendAsync(HubEvents.ChannelClosed, channel);
+            await HubContext.Clients.Clients(clientIds).SendAsync(HubEvents.ChannelClosed, channelId);
         }
 
-        public async Task OnJoinChannel(MemberSummaryResponse member, ChannelSummaryResponse channel)
+        public async Task OnJoinChannel(ChannelSummaryResponse channel, MemberSummaryResponse member)
```

Comment:
currently MemberSummaryResponse member not used

Reply:
removed